src/annotator/sidebar.js

```diff
@@ -18,6 +18,8 @@ import { createShadowRoot } from './util/shadow-root';
  * @typedef {import('../types/bridge-events').SidebarToHostEvent} SidebarToHostEvent
  * @typedef {import('../types/annotator').SidebarLayout} SidebarLayout
  * @typedef {import('../types/annotator').Destroyable} Destroyable
+ * @typedef {'contentContainer'|'createAnnotation'|'fitSideBySide'} DependenciesSidebar
```

Comment:
It isn't immediately obvious when looking at the code in this context what these fields relate to. I think it would be clearer with something like:

```js
/** @typedef {Pick<Guest, ...fields>} SidebarGuest */
/** @typedef {Pick<Guest, ...fields>} BucketBarGuest */

/**
  * @param {SidebarGuest & BucketBarGuest} guest
  */
constructor(...) {
}
```

That said, this kind of thing does add extra complexity to the types, so it is important to have a clear reason for doing this.

In this case we are going to need to make a larger structural change to the code soon to support host frames which are not guest frames. Do you already have a clear plan of how that will relate to these changes? If not I'd hold off on them until that problem is solved.